doc-site/docs/concepts/project-life-cycle.md

```diff
@@ -3,3 +3,27 @@ title: Project Life Cycle
 sidebar_label: Project Life Cycle
 sidebar_position: 2
 ---
+
+## zero init
+The goal of the `init` step is to create the project manifest (`zero-project.yml`).
+
+`zero init` will fetch each **module(s)** defined in the `zero-project.yml`, and prompt user through a series of questions to fill in parameters required by the module definition. At this phase module definition will be parsed and provide defaults and options to guide users to fill in their project details.
+
+:::note
+It's recommended to review the `zero-project.yml` and make adjustments as needed before running `zero create` and `zero apply`
+:::
+
+## zero create
+`zero create` runs on the same folder as `zero-project.yml`, it will scaffold and template out each module `templates/` folder as the basis of your repository, then push them to your version control repository(github).
+
+During scaffolding Zero will also determine based on project's parameters to optionally scaffold conditioned files, for example it will not scaffold the authentication examples if you opted out from this step.
+
+## zero apply
+`zero apply` setups up the final steps of your provisioning, it by default runs the `make` command and modules can specify to change this behavior. Zero expects a two step process of which it will run `check` then `make` to setup the module.
+
+### Check
+`check` is ran for all the modules in your project before attempting to do the `run` step, so if one dependent's `check` fails it will not start the actual provisioning for any of the modules, this is useful to check for missing binaries or API token permissions.
+
+### Apply
+The run step is by default `make` and can be overriden in the module definition. Run should be the one time setup procedures that allow the module to function.
+For example in the infrastructure repository this would be **running terraform**, and for the backend repository this could be **setting up the webhooks for your circleCI to connect to your github**.
```

Comment:
```suggestion
## zero init
The goal of the `init` step is to create the project manifest (`zero-project.yml`).

`zero init` will fetch each **module** defined in `zero-project.yml` from their remote repository, and prompt the user through a series of questions to fill in parameters required by each module. In this phase, the module definition will be parsed and provide defaults, options, and extra context to guide users through filling in their project details.

:::note
It's recommended to review the `zero-project.yml` and make adjustments as needed before running `zero create` and `zero apply`.
:::

## zero create
`zero create` is run in the same folder as `zero-project.yml`. It will template out each module specified in the project manifest as the basis of your repositories, then push them to your version control repository (Github).

During the `create` step, Zero will also conditionally include or exclude certain sets of files, as defined by each module. For example, it will not scaffold the authentication examples if you opted not to use this feature.

## zero apply
`zero apply` is the provisioning step that starts to make real-world changes. By default, it runs a command defined by the module to check for any dependencies, and then runs a command to actually apply any changes.

### Check
`check` is run for all the modules in your project before attempting to do the `run` step, so if a dependent's `check` fails it will not start the actual provisioning for any of the modules. This is useful to check for missing binaries or API token permissions before starting to apply any changes.

### Apply
By default, the run step is to execute `make` in the root of the module, but that can be overridden in the module definition. Run should be the one-time setup commands that allow the module to function.
For example, in the infrastructure repository, this would be to **run terraform**, and for the backend repository, this could be to  **make API calls to CircleCI to set up your build pipeline**.
```